app/controllers/items_controller.rb

```diff
@@ -248,9 +287,15 @@ def update_attributes
     [:publish, :shelve, :preserve].each do |k|
       params[k] = (params[k].nil? || params[k] != 'on') ? 'no' : 'yes'
     end
-    @object.contentMetadata.update_attributes(params[:file_name], params[:publish], params[:shelve], params[:preserve])
+    @object.contentMetadata.update_attributes(
```

Comment:
Same questions around formatting changes that could be addressed outside of this commit/pr